jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/SparklineAssert.java

```diff
@@ -43,27 +28,105 @@ public SparklineAssert labelsPrefixedWith(String prefix) {
+                    label, postfix));
         }
         return this;
     }
 
     @JDIAction("Assert that '{name}' linear gradient map corresponds to the provided map {0}")
     public SparklineAssert linearGradientMap(Map<String, String> gradientColorMap) {
-        jdiAssert(element().getLinearGradientMap(), Matchers.equalTo(gradientColorMap));
+        Map<String, String> actualLinearGradientMap = element().getLinearGradientMap();
+        jdiAssert(actualLinearGradientMap, Matchers.equalTo(gradientColorMap), String.format("Actual linear gradient map " +
+                "'%s' is not equal to expected '%s'", actualLinearGradientMap, gradientColorMap));
         return this;
     }
 
     @JDIAction("Assert that '{name}' SVG path shape is equal to the provided path shape")
     public SparklineAssert pathShapeEqualTo(String pathShape) {
-        jdiAssert(element().getPathShape(), Matchers.equalTo(pathShape));
+        String actualPathShape = element().getPathShape();
+        jdiAssert(actualPathShape, Matchers.equalTo(pathShape), String.format("Actual path shape '%s' is not equal " +
+                "to expected '%s'", actualPathShape, pathShape));
         return this;
     }
 
     @JDIAction("Assert that '{name}' SVG path shape is not equal to the provided path shape")
     public SparklineAssert pathShapeNotEqualTo(String pathShape) {
-        jdiAssert(element().getPathShape(), Matchers.not(pathShape));
+        String actualPathShape = element().getPathShape();
+        jdiAssert(actualPathShape, Matchers.not(pathShape), String.format("Actual path shape '%s' is equal " +
+                "to expected '%s'", actualPathShape, pathShape));
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' has type bar")
+    public SparklineAssert bar() {
+        jdiAssert(element().isBar(), Matchers.is(true), "Element's type is not bar");
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' has type bar")
+    public SparklineAssert trendline() {
+        jdiAssert(element().isTrendline(), Matchers.is(true), "Element's type is not trendline");
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' labels has font size '{0}'")
+    public SparklineAssert labelFontSize(int labelFontSize) {
+        int actualLabelFontSize = element().labelFontSize();
+        jdiAssert(actualLabelFontSize, Matchers.equalTo(labelFontSize), String.format("Element's actual label size '%s' is " +
+                "not equal to '%s'", actualLabelFontSize, labelFontSize));
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' has visible labels")
+    public SparklineAssert labels() {
+        jdiAssert(element().hasLabels(), Matchers.is(true), "Element has not visible labels");
+        return this;
+    }
+
+    @JDIAction("Assert that '{name}' has not visible labels")
+    public SparklineAssert notLabels() {
+        jdiAssert(element().hasLabels(), Matchers.is(false), "Element not labels");
```

Comment:
Please, change an assert description to something more informative like - "Element has visible labels" of "Element has following visible labels:  "